### DISCUSSION TITLE
New pattern for garment cover/carrier

### DISCUSSION THREAD

**Discussion by darigovresearch** [freesewing/freesewing#1620] [Forum]
**What is it that you would like to see happen?**

After making fantastic garments from freesewing you want to protect it either in transit or in deep/winter storage. Commonly known also as a suit cover/dress cover/suit carrier/dress carrier.

**Are you a FreeSewing patron?**

 - [ ] Yes, I am :hugs:
 - [x] No, I am not :thinking:

**Additional context**

Usually have a hole at the top so the top of a coat hanger can be fed through & hung. Can have straps & handles to make them easier to carry. Extra optional pockets for accessories or lavender bags to prevent moths eating your hard work. A long zipper is usually done down the middle but they may be difficult to source, if anyone knows of an inventive way that doesn't require a zip do let us know how below so we can make a variation of the pattern to accommodate potentially.  

Would be nice to have a place to put a label to describe what's inside either on the front or down the side like a shoulder mark. Can be made from upcycled material like torn sail, tarp or tent canvas and can be used as further material to make other freesewing garments if you make a mistake or the cover/carrier becomes worn out.

This will likely be one of our first patterns to see what the contribution process is like & how to generate parametric patterns. Anybody who is new to contributing or interested in helping is welcome to join. See this video if you need some guidance on how you may be able to help - https://www.youtube.com/watch?v=UWA4wyacY2A

Suggested name Gary the garment cover/carrier

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions

**Comment by darigovresearch** [comment]
Reopening discussion from the community call https://github.com/freesewing/freesewing/discussions/5532

**Comment by BenJamesBen** [comment]
Closing because there does not seem to be any interest for active discussion about this topic, per the April 27, 2024 Community Call. https://github.com/freesewing/freesewing/discussions/5724#discussioncomment-9247197

(Please feel welcome to re-open this Discussion if there is interest in resuming active discussion about the topic.)
